docs/quests.proto

```diff
@@ -107,17 +119,24 @@ message QuestState {
   repeated string required_steps = 5;
 }
 
-message QuestStateUpdate {
+message QuestStateWithData {
   string quest_instance_id = 1;
   string name = 2;
   string description = 3;
   QuestState quest_state = 4;
 }
 
+message QuestStateUpdate {
+  QuestStateWithData quest_data = 1;
+  string event_id = 2;
+}
+
 message UserUpdate {
   oneof message {
-    QuestStateUpdate quest_state = 1;
-    fixed32 event_ignored = 2;
+    QuestStateUpdate quest_state_update = 1;
+    QuestStateWithData new_quest_started = 2;
+    QuestStateWithData current_quest_state = 3;
```

Comment:
What's the difference between `quest_state_update` and `current_quest_state` ?

Reply:
`current_quest_state` is when the user just subscribed and receives the current state of their current active quests. These states are not related to an event id reaction as `quest_state_update `.